ipywidgets/tests/.tsdrc

```diff
@@ -0,0 +1,3 @@
+{
+	"token": "94e9b622a27d97abbd3775256c684f39b940a2c0"
```

Comment:
It increases the Github checkout rate limit from a low 60ish to 5000 or so per hour.  This is for TSD, alternatively I can check the .d.ts files in (typescript type definitions).